Radzen.Blazor/RadzenDatePicker.razor.cs

```diff
@@ -249,6 +251,35 @@ protected override void OnInitialized()
         [Parameter]
         public string InputClass { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Minimum Selectable Date.
+        /// </summary>
+        /// <value>The Minimum Selectable Date.</value>
+        [Parameter]
+        public DateTime? MinimumDate { get; set; }
```

Comment:
Maybe these properties can be named simply Min and Max similar to other components like RadzenNumeric, RadzenSlider, etc .

Reply:
MinDate and MaxDate or just Min/Max?

Reply:
Just Min/Max are good enough in my opinion - will match other Radzen components with min/max settings.